iOS/FVM/FVM/Highlighting/MaterialProcessor.swift

```diff
@@ -56,4 +48,12 @@ internal class MaterialProcessor {
     internal func hasKeyFor(_ key: String) -> Bool {
         return materialStore.keys.contains(key)
     }
+    
+    private func setHighlightedMaterial(for nodeName: String) {
```

Comment:
Changed to private since this is not called from anywhere else and its purpose is rather private